package/rpi-fbcp/S11fbcp

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+ 
```

Comment:
Yep, I know... I think I will bother you with this stuff to get informations about how the configuration works from recalbox.conf to config file such config.txt :)